packages/editor/src/store/reducer.js

```diff
@@ -412,6 +412,27 @@ export function postSavingLock( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Post autosaving lock.
+ *
+ * When post autosaving is locked, the post will not autosave.
+ *
+ * @param {PostAutosavingLockState} state  Current state.
+ * @param {Object}                  action Dispatched action.
+ *
+ * @return {PostLockState} Updated state.
+ */
+export function postAutosavingLock( state = {}, action ) {
```

Comment:
it needs to be included in the export statement at the end of the file as well. This should resolve all failing tests.

Reply:
ah, right - thanks! added.